node/entry/entry_cli.go

```diff
@@ -97,7 +102,10 @@ P2P traffic are encrypted by %[1]s. Every tunnel has different secret key.
 			}
 
 			// Normalize the gateway scheme
-			gateway := config.APIGateway()
+			gateway := constant.DefaultAPIGateway
```

Comment:
It's better to keep compatible with environment variables and make the command line parameter has highest priority.